cmd/terramate/cli/cli.go

```diff
@@ -580,6 +588,31 @@ func hasVendorDirConfig(cfg hcl.Config) bool {
 	return cfg.Vendor != nil && cfg.Vendor.Dir != ""
 }
 
+func (c *cli) triggerStack() {
+	stack := c.parsedArgs.Experimental.Trigger.Stack
+	reason := c.parsedArgs.Experimental.Trigger.Reason
+	if reason == "" {
+		reason = "manually created through terramate CLI"
+	}
+	logger := log.With().
+		Str("stack", stack).
+		Logger()
+
+	logger.Debug().Msg("creating stack trigger")
+
+	if !path.IsAbs(stack) {
+		logger.Fatal().Msg("stack must be a Unix absolute path, like /stack")
```

Comment:
```suggestion
		logger.Fatal().Msg("stack must be a project absolute path, like /stack")
```
Saying *UNIX* can be confused with a host path.